### DISCUSSION TITLE
Feature Request - Allow wiki edit access for specified users

### DISCUSSION THREAD

**Discussion by danielhjacobs** [community/community#10376] [Other Feature Feedback, Questions, & Ideas]
The only options for wiki editing are everyone or collaborators only. However, some organizations may have helpful non-collaborators to whom they'd want to grant edit access without allowing edit access to everyone. Trolls can be blocked on a case-by-case basis, but sometimes that may prove too obnoxious. I propose allowing up to ~20 users to be added to a list of other allowed editors, so with the collaborator only setting, only collaborators or anyone on that list can edit the wiki.

**Comment by rusefillc** [comment]
+1 please

current way of matching push access is too limited

See https://github.com/OpenBoard-org/OpenBoard/issues/548

**Reply by mck1117** [reply]
Exhibit A: https://github.com/lettuce-io/lettuce-core/issues/2002#event-6036761003

**Reply by danielhjacobs** [reply]
Exhibit B: https://github.com/ruffle-rs/ruffle/wiki/Frequently-Asked-Questions-For-Users/_compare/f5ede5596c863b856d578dba6e2fec9b6d230285...890e801df9a752176152a7eb8213202296205c11 (the updated link was malware but I believe has since been removed). Allowing anyone to update the wiki opens the wiki up to becoming a home for malware, but collaborators only is incredibly strict.
